blog/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from blog.views import PostListView, PostDetailView, CommentCreateView
+
+app_name = "blog"
+
+urlpatterns = [
+    path("", PostListView.as_view(), name="index"),
+    path("posts/<int:pk>/", PostDetailView.as_view(), name="post-detail"),
+    path(
+        "comments/<int:pk>/create",
```

Comment:
url comments/pk/create looks not logical. Maybe posts/pk/create-comment or smth like this?